src/metabase/db/custom_migrations.clj

```diff
@@ -1157,6 +1157,10 @@
 (defn- internal-user-exists? []
   (pos? (first (vals (t2/query-one {:select [:%count.*] :from :core_user :where [:= :id config/internal-mb-user-id]})))))
 
+(define-migration CreateInternalUserNoOp
```

Comment:
I couldn't see where this migration is triggered, do you still need it?

Reply:
good call, removed.